app/views/provider/admin/cms/templates/_quick_links.html.erb

```diff
@@ -14,8 +14,8 @@
     <li>
       <p>
         <%= icon('question-circle fa-li') %>
-        Follow the
-        <%= link_to 'Quickstart guide', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
+        Creating the
```

Comment:
```suggestion
        Check out documentation on
```

or whatever similar text so it's clear what the link is about

Reply:
Ah... I understand...